Tests/ConnectLibraryTests/buf.gen.yaml

```diff
@@ -7,6 +7,7 @@ plugins:
     opt: >
       GenerateAsyncMethods=true,
       GenerateCallbackMethods=true,
+      GenerateServiceMetadata=true,
```

Comment:
I'm guessing stripping and what not will make generating this maybe fine by default?

Reply:
This is `false` by default 😃 https://github.com/bufbuild/connect-swift/pull/86/files#diff-851cb9abd0a1e3e867e27157177f8eac6681cf46ceeac20489101ebeb0a43547R80

Reply:
Oh I meant to suggest that we make it true by default instead.

Reply:
Sorry, edited my comment (probably while you were replying) - for now I think we can bias towards reducing generated code. This way if we want to change the default in the future, it's an additive rather than breaking change

Reply:
Keep in mind that whatever's off by default will effectively be unavailable to users of any future BSR-generated Swift packages. I'd imagine that these constants are most likely just a few kB - is default off the right choice for most users?

Reply:
That is true. I can make it `true` by default.